grafana_inviter/cli.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+"""Script for fetching users from LDAP and sending/generating invites for Grafana access.
+"""
+
+import argparse
+from getpass import getpass
+import sys
+import anyconfig
+
+from .accounts import AccountManager
+from .grafana import Grafana
+
+
+def parse_args(args):
+    """Returns parsed commandline arguments.
+    """
+
+    parser = argparse.ArgumentParser(description="Script for generating/sending Grafana invite URLs. By default only invite URLs are generated.")
+    parser.add_argument("--ldap-url", type=str, required=True, help="LDAP URL")
+    parser.add_argument("--ldap-user", type=str, required=True, help="LDAP service account username")
+    parser.add_argument("--ldap-password", type=str, help="LDAP service account password")
+    parser.add_argument("--grafana-url", type=str, required=True, help="Grafana URL")
+    parser.add_argument("--grafana-token", type=str, help="Grafana API token")
+    parser.add_argument("--send-invite-mail", action="store_true", help="Send mail with Grafana invite URL.")
+    parser.add_argument("--config", type=str, help="Location of configuration file")
+    return parser.parse_args(args)
+
+
+def main():
+    """Main entrypoint
+    """
+
+    args = parse_args(sys.argv[1:])
+
+    config = None
```

Comment:
Should allow for pretty much all of the configuration options in the CLI to be pulled from here if not specified in the arguments.